### ISSUE TITLE
[Feature Request] Additional CSS Identifiers for Default Templates

### ISSUE THREAD

**Issue by thepsion5** [philiplb/CRUDlex#40]
# Why?

For most of my intended usage, any changes to the look and feel of the default templates can be accomplished via CSS. However, there are cases where there are no identifiers that can be used to differentiate between an entity's list table or an entity's show table, for example. Adding a few would make using custom CSS much simpler. While using custom templates is possible, it would result in a lot of code duplication just to change a single line.

# Proposed Template Class/ID Additions

1. Adding a `crudEntityListTable` class to the primary table in `list.twig`
2. Adding class `crudEntityShowTable` to the primary table in `show.twig`
3. Adding class `crudEntityChildListTable` to the child table in `show.twig`
4. Adding ID `crudChild{{ childEntity.get(child[0]) }}` to the child table in `show.twig`

# While We're at it...

When working with the standard CRUD application UI, I frequently alter the width of columns to make things look a bit cleaner. Currently, this can only be accomplished by using an nth-child CSS selector, which breaks if you change the order of the columns. Instead, adding an ID for each column based on the field name would make this a bit easier (in `list.twig`):

```
    {% for field in fields %}
        <th id="crudEntityField{{ field }}">
        (snip)
        </th>
    {% endfor %}
    <th id="crudA">Action</th>
```

But I would consider this less important.
         
I'm not attached to any of the specific names, I just tried to pick names that were consistent with your other classes. Also happy to submit a PR with said changes as well.

**Comment by philiplb** [comment]
Now that's a nice ticket.
Will be done at some point today. :)

**Comment by philiplb** [comment]
Resolved in 88a060fe1ff9632be9c1705c9b1b821a4d540d89
I did two adjustements as it could have ended up with invalid/multiple ids in the original proposal:

```
   {% for i, child in children %}
       <h2>{{ child[0] }}</h2>
       <table class="table table-striped crudEntityChildListTable" id="crudChild{{ i }}">
```

```
    {% for i, field in fields %}
        <th id="crudEntityField{{ i }}">
        (snip)
        </th>
    {% endfor %}
    <th id="crudA">Action</th>
```
